yarn.lock

```diff
@@ -442,7 +442,20 @@ babel-eslint@^7.2.3:
     babel-types "^6.23.0"
     babylon "^6.17.0"
 
-babel-generator@^6.18.0, babel-generator@^6.24.1, babel-generator@^6.26.0:
+babel-generator@^6.18.0:
```

Comment:
Did you edit lockfile by hand? This change is weird, `^6.18.0` should resolve to the same that `^6.24.1` and `^6.26.0` below

Reply:
oh, thanks!

Reply:
(running `yarn-tools` to dedupe should be fine, fwiw)